src/dune_rules/coq_stanza.ml

```diff
@@ -145,6 +149,8 @@ module Theory = struct
        and+ modules = Stanza_common.modules_field "modules"
        and+ enabled_if = Enabled_if.decode ~allowed_vars:Any ~since:None ()
        and+ buildable = Buildable.decode in
+       (* boot libraries cannot depend on other theories *)
+       if boot && List.is_non_empty buildable.theories then boot_has_deps buildable;
```

Comment:
The location of `List.hd buildable.theories` should be a bit more useful to the user in this error.

Reply:
Done thanks.